lib/scr/bloc/dice_bloc.dart

```diff
@@ -0,0 +1,47 @@
+import 'dart:async';
+import 'dart:math';
+
+import 'i_dice_bloc.dart';
+
+class DiceBloc extends IDiceBloc {
+  final int _numberOfDice = 4;
```

Comment:
The number of dices that you used can be a constant

Reply:
Fixed